tests/kubeless/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:latest
```

Comment:
Please do not use latest versions, see also https://github.com/kyma-project/community/blob/master/guidelines/docker-images.md